CONTRIBUTING.md

```diff
@@ -201,7 +202,7 @@ Ports have a weaker contribution bar, at least initially. A functionally correct
 
 #### Copying Files from Other Projects
 
-.NET Core uses some files from other projects, typically where a binary distribution does not exist or would be inconvenient.
+.NET uses some files from other projects, typically where a binary distribution does not exist or would be inconvenient.
```

Comment:
```suggestion
.NET runtime uses some files from other projects, typically where a binary distribution does not exist or would be inconvenient.
```

Reply:
Similar to above adding "the" makes it read better IMO:

```diff
- .NET runtime
+ The .NET runtime
```